src/rtc-to-net/rtc-to-net.ts

```diff
@@ -175,7 +165,7 @@ module RtcToNet {
       });
       var writeSnapshot = () => {
         this.getSnapshot().then((snapshot:RtcToNetSnapshot) => {
-          log.info('snapshot: %1', [JSON.stringify(snapshot)]);
+          log.info('snapshot: ' + JSON.stringify(snapshot));
```

Comment:
I don't see how it'll eventually help. FYI, @jpevarnek made some recent improvements and now you could also just do:
```typescript
 log.info('snapshot: %1', snapshot);
```

Or even just:
```typescript
 log.info(snapshot);
```